src/macros.rs

```diff
@@ -25,11 +25,14 @@ macro_rules! public_test_dep {
 /// platforms need and elsewhere in this library it just looks like normal Rust
 /// code.
 ///
-/// When the weak-intrinsics feature is enabled, all intrinsics functions are
-/// marked with #[linkage = "weak"] so that they can be replaced by another
-/// implementation at link time. This is particularly useful for mixed Rust/C++
-/// binaries that want to use the C++ intrinsics, otherwise linking against the
-/// Rust stdlib will replace those from the compiler-rt library.
+/// All intrinsics functions are marked with #[linkage = "weak"] when
+/// `not(windows) and not(target_vendor = "apple")` except `mem` intrinsics, which
+/// will only have the `#[linkage = "weak"]` attribute when the `mem-weak-intrinsics`
+/// feature is enabled.
```

Comment:
`mem-weak-intrinsics` is not needed since the root cause of the original issue was tracked down to https://github.com/Kingcom/armips/pull/242.

Reply:
yup, removed it. Looks like I didn't commit the changes to `Cargo.toml` before